docs/2-advanced/01-configuration.md

```diff
@@ -60,6 +58,14 @@ import { setTheme, getDefaultTheme } from "@ui5/webcomponents-base/dist/config/T
 setTheme(getDefaultTheme());
 ```
 
+**Note: Deprecated themes**
+
+The following themes are deprecated and no longer developed. We recommend using `Quartz` (sap_fiori_3) and `Horizon` (sap_horizon) theme families.
+- The `sap_belize` is known as `Belize` [deprecated since 1.21].
+- The `sap_belize_hcb` is known as `High Contrast Black` [deprecated since 1.21].
+- The `sap_belize_hcw` is known as `High Contrast White` [deprecated since 1.21].
+* the themes will work until the next major version (2.0) and will be removed afterwards.
```

Comment:
Remove * or transform it to `**note**` otherwise this lien will appear as list item.